src/biokbase/auth.py

```diff
@@ -93,30 +101,14 @@ def get_agent_token(login_token, token_name="NarrativeAgent"):
     created: ms since epoch
     """
     headers = {"Authorization": login_token, "Content-Type": "Application/json"}
-    data = json.dumps({"name": token_name})
-    r = requests.post(token_api_url + endpt_token, headers=headers, data=data)
+    data = {"name": token_name}
+    r = requests.post(token_api_url + endpt_token, headers=headers, json=data)
     if r.status_code != requests.codes.ok:
         r.raise_for_status()
-    agent_token_info = json.loads(r.content)
+    agent_token_info = TokenInfo(r.json())
     return agent_token_info
 
 
-def revoke_token(auth_token, revoke_id):
```

Comment:
not used? not needed?

Reply:
Right, I forgot to note that. Both - it's not used, not needed, and I don't think that's a valid endpoint.